frontend/helpers/issues.ts

```diff
@@ -15,34 +15,50 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { get, omit } from 'lodash';
+import { get } from 'lodash';
 import { getAPIUrl } from '../services/api';
 import { STATUSES_COLOURS, STATUSES_ICONS, STATUSES } from '../constants/issues';
 import { isAdmin, hasPermissions, PERMISSIONS } from './permissions';
-import { sortByDate } from './sorting';
+
+const renameFieldIfExists = (issue, fieldName, newFieldName) => {
+	if (issue[fieldName]) {
+		issue[newFieldName] = issue[fieldName];
+	}
+};
 
 export const prepareIssue = (issue, jobs = []) => {
-	const thumbnail = getAPIUrl(issue.thumbnail);
-	const descriptionThumbnail = issue.viewpoint && issue.viewpoint.screenshot
+	if (issue.thumbnail) {
```

Comment:
If you really need to split the code to all these statements, it's better to spread the issue once (at the beginning) and then add/update fields. You'll avoid unnecessary object spreading on each if statement.

```
const preparedIssue = { ...issue };

if (preparedIssue.thumbnail) {
    preparedIssue.thumbnail = getAPIUrl(preparedIssue.thumbnail);
}

return preparedIssue;
```